src/ol3-layerswitcher.js

```diff
@@ -197,6 +197,11 @@
                 input.name = 'base';
             } else {
                 input.type = 'checkbox';
+                var rsl = this.getMap().getView().getResolution();
+                if (rsl > lyr.getMaxResolution() || rsl < lyr.getMinResolution()){
+                    li.className += ' disabled';
+                    input.disabled = true; // css only solution not working in IE
```

Comment:
I think I'd rather limit the scope to just indicating if a layer is currently within view and allow users to toggle visibility even when it's out of range. Therefore I'd prefer not disabling the input and removing the `pointer-event: none` in `src/ol3-layerswitcher.css`